src/layouts/Files.jsx

```diff
@@ -19,6 +20,7 @@ export default class Files extends Component {
       files: [],
       chosenFile: null,
       pendingFileUpload: null,
+      shouldRedirect: false,
```

Comment:
What situation are we expecting to redirect to `Not Found` for files and images? When the entire directory doesn't exist?

Reply:
As discussed offline, this was to prevent users from accessing any route in a site they did not have access to. No longer necessary after the rework in https://github.com/isomerpages/isomercms-frontend/pull/280/commits/29101878097a4e7a4c7a80df5dec76b18b9cb6e0